util/typeutil/stringInt.go

```diff
@@ -0,0 +1,32 @@
+package typeutil
```

Comment:
This might be a better fit for the existing `jsonutil` package as it's a helper for json encoding.

Reply:
this is done - https://github.com/prebid/prebid-server/pull/2617/commits/cad0d8555f0ae53f362d09c6f837cebb5c9ab707